snooty/rstspec.toml

```diff
@@ -109,21 +109,23 @@ example = """.. tip:: ${1:Title}
    ${2:Tip content}
 """
 
-[directive.versionchanged]
-argument_type = "string"
+[directive._baseversiondirective]
+argument_type = {type = "string", required = true}
 content_type = "block"
+
+[directive.versionchanged]
+inherit = "_baseversiondirective"
 example = """.. versionchanged:: ${1:v1.0}
    ${2:Describe what changed in that version. Description is optional.}
 """
 
 [directive.versionadded]
-argument_type = "string"
-content_type = "block"
+inherit = "_baseversiondirective"
 example = """.. versionadded:: ${1:v1.0}
    ${2:Describe what was added in that version. Description is optional.}
 """
 [directive.deprecated]
-argument_type = "string"
+argument_type = {type = "string", required = false}
 content_type = "block"
 example = """.. deprecated:: ${1:v1.0}
```

Comment:
If you were keen, you could indicate that the version is optional in this example string.

Reply:
I don't actually really understand the example string format -- could I have an example please? :D

Reply:
How is this?